core/src/Telemetry/XMsTelemetryInfo.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Identity.Core.Telemetry
+{
+    internal class XMsTelemetryInfo
```

Comment:
Please add comments to the class and public methods / properties where they are not self explanatory (XMSTelemetryInfo is not, SpeInfo is not, Version is quite overloaded with meaning)

Reply:
Done